pkgs/by-name/ma/maim/package.nix

```diff
@@ -52,6 +53,15 @@ stdenv.mkDerivation rec {
     icu
   ];
 
+  patches = [
+    # Use C++17 as required by icu
+    (fetchpatch {
+      name = "maim-c++-17.patch";
+      url = "https://github.com/naelstrof/maim/commit/e7fe09b6734baeb59081b8805be542c92178cf0f.patch";
+      sha256 = "0z9zvrr8msfli88jmhxm5knysi385s48j2w7zpacc7qhf4c5zh8c";
```

Comment:
Should probably use `hash = ""`; with an SRI hash instead of using `sha256 = ` directly

Reply:
No, that makes no difference.

Reply:
Alright, I don't care that much.